install/linux/usr/share/odemis/sim/secom2-sem-ctrl-sim.odm.yaml

```diff
@@ -0,0 +1,221 @@
+    }
+}
+
+"EBeam control": {
+    creator: "SEM via API",
+    role: null,
+    init: {
+        hfw_nomag: 0.114,  # m, theoretical HFW if mag == 1x
+    },
+}
+
+"EBeam Focus": {
+    # Internal child of zeiss.SEM, so no class
+    role: ebeam-focus,
+}
+
+"Optical Objective": {
+    class: static.OpticalLens,
+    role: lens,
+    init: {
+       mag: 40.0, # ratio, (actually of the complete light path)
+       na: 0.95, # ratio, numerical aperture
+       ri: 1.0, # ratio, refractive index
+    },
+    affects: ["Camera"]
+}
+
+# Semrock FF01-440/521/607/700
+"Optical Emission Filter": {
+    class: static.LightFilter,
+    role: filter,
+    init: {band: [[420.e-9, 460.e-9],
+                  [510.e-9, 532.e-9],
+                  [590.e-9, 624.e-9],
+                  [677.e-9, 723.e-9]]}, # m,m
+    affects: ["Camera"]
+}
+
+# Axes: X is horizontal on screen (going left->right), physical: far->close when looking at the door
+#       Y is vertical on screen (going bottom->top), physical: left->right when looking at the door
+"Camera": {
+    class: andorcam2.FakeAndorCam2,
+    role: ccd,
+    init: {
+        device: 0,
+        image: "andorcam2-fake-clara.tiff",
+    },
+    properties: {
+        exposureTime: 0.3, # s
+        # targetTemperature: -40, # °C
+#        resolution: [2048, 2048],  # To crop Y axis
+    }
+}
+
+# Use 3 MultiplexActuators to separate the axes of the PIGCS over different roles
+
+# Axes should be synchronized with the camera
+# Stage axes are moving the sample, so they should opposite direction than convention
+# (so the stage "position" is the current position observed)
+"Sample Stage": {
+    class: actuator.MultiplexActuator,
+    role: stage,
+    dependencies: {"x": "Stage Actuators", "y": "Stage Actuators"},
+    init: {
+        axes_map: {"x": "sx", "y": "sy"},
+    },
+    affects: ["Camera", "SEM E-beam"],
+}
+
+"Objective Stage": {
+    class: actuator.MultiplexActuator,
+    role: align,
+    dependencies: {"x": "Stage Actuators", "y": "Stage Actuators"},
+    init: {
+        axes_map: {"x": "ox", "y": "oy"},
+    },
+    affects: ["Camera"],
+}
+
+# Axis Z: (Physical) bottom->top when looking at the door
+#         (On screen) bottom->top (so distance to sample is smaller when going up)
+"Optical Focus": {
+    class: actuator.MultiplexActuator,
+    role: focus,
+    dependencies: {"z": "Stage Actuators"},
+    init: {
+        axes_map: {"z": "oz"}
+    },
+    affects: ["Camera"],
+}
+
+# Network should be set up and the C-863 should be configured for DHCP.
+# Each E-861 should be configured for the right stage.
+# PI E-861 addresses and stage configuration:
+# optical X = 1 (open loop)
+# optical Y = 2 (open loop)
+# optical Z = 3 (closed loop)
+# stage X = 4 (closed loop)
+# stage Y = 5 (closed loop)
+
+"Stage Actuators": {
+    class: pigcs.FakeBus,
+    role: null,
+    init: {
+        port: "/dev/fake",
+        # axis -> controller, channel, closed-loop?
```

Comment:
Why is the closed-loop with a "?"? Can it be removed or a comment be added to clarify?